src/Webhook/Services/ChargeOrderService.php

```diff
@@ -413,6 +498,10 @@ public function prepareReturnMessage(ChargeInterface $charge)
             return $returnMessage;
         }
 
+        if ($charge->getStatus()->equals(ChargeStatus::failed())) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Mundipagg\Core\Kernel\ValueObjects\ChargeStatus' in method 'prepareReturnMessage'.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)